src/components/AddComment.jsx

```diff
@@ -0,0 +1,72 @@
+import { useState } from "react";
+import { useContext } from "react";
+import { UserContext } from "../context/UserContext";
+import { useParams } from "react-router-dom";
+import postComment from "../utils/postComment.utils";
+
+function AddComment() {
+  const [inputComment, setInputComment] = useState("");
+  const [isError, setIsError] = useState(false);
+  const [newestComment, setNewestComment] = useState(null);
+  const { user } = useContext(UserContext);
+  const { review_id } = useParams();
+
+  function handleSubmit(event) {
+    event.preventDefault();
+
+    if (inputComment.trim() === "") {
+        setIsError(true)
+        return
+    }
+
+    const newComment = {
+      body: inputComment,
+      author: user,
+    };
+    setInputComment("");
+    postComment(newComment, review_id)
+      .then((response) => {
+        const comment = response.comment;
+        setNewestComment(comment);
+        setIsError(false)
+      })
+      .catch((error) => {
+        setIsError(true);
+      });
+  }
+
+  return (
+    <main>
+      <form>
+        <input
```

Comment:
I'd consider changing this for a text area so that a user can add several lines of comment rather than one :)